OpenRA.Game/ExternalModSwitcher.cs

```diff
@@ -0,0 +1,56 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace OpenRA
+{
+	public class ExternalModSwitcher
```

Comment:
Can this be merged into the `ExternalMods` class and then called as `Game.ExternalMods.Switch()`?

Reply:
Thought about that too, was not sure about that so started with the dedicated class.

Reply:
Problem is that we need to supply `launchWrapper` , we could supply it through the constructor, but we use  ExternalMods  in other places where we dont have that and also not using it.